obal/data/roles/repoclosure/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: 'Build additional repos'
+  set_fact:
+    additional_repos: "{{ additional_repos|default([]) + ['--repofrompath=' + item.name + ',' + item.url] }}"
+  with_items: "{{ repoclosure_additional_repos }}"
+  when: repoclosure_additional_repos is defined
```

Comment:
Isn't it always defined, as it's in `defaults/main.yml`?

Reply:
Truth. Fixed.